frontend/appflowy_flutter/packages/flowy_infra/lib/plugins/service/location_service.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+
+import 'package:path_provider/path_provider.dart';
+
+class PluginLocationService {
+  static Future<Directory> get fallback async =>
+      await getApplicationDocumentsDirectory();
```

Comment:
I think we should place the theme files in appflowy data folder. check out the `appFlowyDocumentDirectory`

```dart
Future<Directory> appFlowyDocumentDirectory() async {
  switch (integrationEnv()) {
    case IntegrationMode.develop:
      final Directory documentsDir = await getApplicationSupportDirectory()
        ..create();
      return Directory(path.join(documentsDir.path, 'data_dev')).create();
    case IntegrationMode.release:
      final Directory documentsDir = await getApplicationSupportDirectory();
      return Directory(path.join(documentsDir.path, 'data')).create();
    case IntegrationMode.test:
    case IntegrationMode.integrationTest:
      return Directory(path.join(Directory.current.path, '.sandbox'));
  }
}
```